src/components/Hits.vue

```diff
@@ -3,7 +3,10 @@
     v-if="state"
     :class="suit()"
   >
-    <slot :items="items">
+    <slot
+      :items="items"
+      :insights="state.insights"
```

Comment:
We could also add the function to the `item` slot. It would avoid to recreate the full UI to use insights.

Reply:
added here: https://github.com/algolia/vue-instantsearch/commit/80446a6196168cedba999cff0121b875c94cf836